Makefile

```diff
@@ -125,15 +129,65 @@ test: examples test-stable
 analyse-stack-sizes:
 	cargo stack-sizes $(release) --example $(EXAMPLE) $(features) -- -Z emit-stack-sizes
 
+
+
 .PHONY: apollo3
 apollo3:
 	LIBTOCK_PLATFORM=apollo3 cargo run --example $(EXAMPLE) $(features) \
-		--target=thumbv7em-none-eabi $(release)
+		--target=thumbv7em-none-eabi $(release) --
 	mkdir -p target/tbf/apollo3
 	cp target/thumbv7em-none-eabi/release/examples/$(EXAMPLE).tab \
 		target/thumbv7em-none-eabi/release/examples/$(EXAMPLE).tbf \
 		target/tbf/apollo3
 
+# ============================================
+# Esp32-C3 DevKitM 1
+# ============================================
```

Comment:
I had the per-board targets in alphabetical order, would you mind putting them back into alphabetical order?

(The usage string at the top of the file should be in alphabetical order as well.)